go.mod

```diff
@@ -81,4 +81,4 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 )
 
-replace github.com/jmoiron/sqlx v1.2.0 => github.com/mfateev/sqlx v0.0.0-20180910213730-fa49b1cf03f7
+replace github.com/jmoiron/sqlx v1.2.0 => github.com/longquanzheng/sqlx v0.0.0-20191125235044-053e6130695c
```

Comment:
please add a comment / readme somewhere explaining why this is needed and also create a github issue so that we don't forget about this

Reply:
Yes I have created an issue. I will add a comment